apprentice-action/tests/endpoint.test.js

```diff
@@ -0,0 +1,42 @@
+const axios = require('axios');
+const core = require('@actions/core');
+const expect = require('chai').expect
+
+describe('Tests to the "/" endpoint', () => {
+    it('should return a 200 status code', done => {
+        axios('http://localhost:80/').then(res => {
```

Comment:
same for these tests - instead of using `done`, we can mark the anonymous functions as `async`, and use `await` on the calls to `axios`.